src/XMakeTasks/UnitTests/GenerateResource_Tests.cs

```diff
@@ -2071,6 +2071,7 @@ public void SourceItemMissing()
 #else
         [Fact(Skip = "https://github.com/Microsoft/msbuild/issues/297")]
 #endif
+        [PlatformSpecific(Xunit.PlatformID.Windows)]
```

Comment:
IIRC the state file's purpose was to record any links to files from the resx, so that if the resx timestamp has not changed we can use the state file to get the list of other files to timestamp check. If that's the case then it's not relevant to xplat as resx there can't link files and the code in GR for that is ifdeffed out.

Also it relies on binary serialization which first appears in a corefx version post 1.1 so I suspect it's not available to you.